lib/secure_headers/headers/content_security_policy_config.rb

```diff
@@ -55,12 +53,10 @@ def directive_value(directive)
       end
     end
 
-    def modified?
-      @modified
-    end
-
     def merge(new_hash)
-      ContentSecurityPolicy.combine_policies(self.to_h, new_hash)
+      new_config = self.dup
```

Comment:
@oreoshake - I know we had chatted about the expectations of `merge`. But, given that `merge!` was overwriting existing values, it seems like `merge` should to the same (or `merge!` should do the opposite). It seems like `append` should be used for any kind of "combine these policies" use cases.

Reply:
I think we should move as far away from treating this like a hash as possible. Using verbs like append/combine/overwrite would be clearer. But we'll have to alias and deprecate first.